deploy/scripts/salt-master-init.sh

```diff
@@ -90,7 +90,7 @@ saltmaster_bootstrap() {
     log_info "Salt master, minion setup (salt-master-setup.sh)"
     test -n "$MASTER_HOSTNAME" || exit 1
 
-    pgrep salt-master | xargs -i{} $SUDO kill -9 {}
+    killall -9 salt-master
```

Comment:
I would use `pkill` instead (eg. to be compatible with Solaris :D) and do the same for salt-minion to be unified. You should also use `$SUDO`